target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -146,6 +146,12 @@ case "$FIRMWARE" in
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
 		;;
+	comfast,cf-ew72)
+		caldata_extract "art" 0x5000 0x2f20
```

Comment:
You would use
ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary art 0x0) +2)
here after our new information, and thus can just merge this case with the previous.

Reply:
Okay, I think I get it now.  I'm just going to collapse with the above then since now it's identical to the CF-E560AC.